src/shims/unix/fd.rs

```diff
@@ -189,6 +189,26 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         }
     }
 
+    fn ioctl(&mut self, args: &[OpTy<'tcx>]) -> InterpResult<'tcx, Scalar> {
+        let this = self.eval_context_mut();
+
+        let [fd_num, cmd] = check_min_arg_count("ioctl", args)?;
+        let fioclex = this.eval_libc_u64("FIOCLEX");
+
+        let fd_num = this.read_scalar(fd_num)?.to_i32()?;
+        let cmd = this.read_scalar(cmd)?.to_u64()?;
```

Comment:
```suggestion
        let fioclex = this.eval_libc_u64("FIOCLEX");

        let [fd_num, cmd] = check_min_arg_count("ioctl", args)?;
        let fd_num = this.read_scalar(fd_num)?.to_i32()?;
        let cmd = this.read_scalar(cmd)?.to_u64()?;
```
Let's separate the steps of parsing our arguments, and evaluating some global constants.